pkg/eventsapi/event_versions.go

```diff
@@ -0,0 +1,17 @@
+package eventsapi
+
+type EventVersion int
```

Comment:
This may be easier if we just make this a `string` and set the constants as `v1` and `v2`.   It's a little strange that the integers start at `0` in the current implementation.

Reply:
Fixed here: 438964d